### PR TITLE
Simplyfy move and redeem utilities

### PR DESCRIPTION
**Pull request by kasiastreich**

### CODE REVIEW

**Comment by roger3cev** [comment]
build

File: build.gradle [hunk 1]
```diff
@@ -68,6 +68,8 @@ subprojects {
         maven { url "https://repo.gradle.org/gradle/libs-releases-local/" }
     }
 
+    apply plugin: 'kotlin'
```
Comment (roastario) [inline build.gradle]: y?

File: workflows/src/integrationTest/kotlin/src/test/kotlin/com/r3/corda/lib/tokens/integrationTest/TokenDriverTest.kt [hunk 2]
```diff
@@ -63,40 +66,47 @@ class TokenDriverTest {
...
-            val dvpTx = nodeA.rpc.startFlowDynamic(
-                    DvPFlow::class.java,
+            println("FOUR HALF")
+            val dvpTx = nodeA.rpc.startFlow(
+                    ::DvPFlow,
                     house,
                     nodeBParty
             ).returnValue.getOrThrow()
-
+            println("FIVE")
```
Comment (roastario) [inline workflows/src/integrationTest/kotlin/src/test/kotlin/com/r3/corda/lib/tokens/integrationTest/TokenDriverTest.kt]: we ok leaving printlns ?

Comment (roger3cev) [reply workflows/src/integrationTest/kotlin/src/test/kotlin/com/r3/corda/lib/tokens/integrationTest/TokenDriverTest.kt]: whoop swill remove

**Comment by roger3cev** [comment]
@roastario over to you

File: modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/House.kt [hunk 3]
```diff
@@ -15,5 +17,11 @@ data class House(
         val valuation: Amount<FiatCurrency>,
         override val maintainers: List<Party>,
         override val fractionDigits: Int = 5,
-        override val linearId: UniqueIdentifier = UniqueIdentifier()
-) : EvolvableTokenType()
\ No newline at end of file
+        override val linearId: UniqueIdentifier
+) : EvolvableTokenType()
+
+class Test : Contract {
```
Comment (kasiastreich) [inline modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/House.kt]: What is it?

Comment (roger3cev) [reply modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/House.kt]: was a test to see if the absence of it was messing things upp... removed it.

**Comment by roger3cev** [comment]
@kasiastreich over to you!

**Comment by roger3cev** [comment]
build

**Pull request merged**